scipy/optimize/_differentialevolution.py

```diff
@@ -135,18 +134,28 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``np.std(pop) <= atol + tol * np.abs(np.mean(population_energies))``,
         where and `atol` and `tol` are the absolute and relative tolerance
         respectively.
-    aggressive : bool, optional
-        The best solution vector is continuously updated within a single
-        generation. This can lead to faster convergence as trial vectors can
-        take advantage of continuous improvements in the best solution.
+    updating : str, optional
+        Should be one of:
+
+            - 'immediate'
+            - 'deferred'
+
+        If `immediate` the best solution vector is continuously updated within
+        a single generation. This can lead to faster convergence as trial
+        vectors can take advantage of continuous improvements in the best
+        solution.
```

Comment:
add a reference or explain more in Notes?

Reply:
There is a reference [4] in the notes, but I've added here as well.